test/integration/consul-container/test/gateways/http_route_test.go

```diff
@@ -33,10 +32,13 @@ func randomName(prefix string, n int) string {
 }
 
 func TestHTTPRouteFlattening(t *testing.T) {
+	t.Skip()
```

Comment:
Is this intentional?

Reply:
Ah... this is probably why the tests were 'suddenly working in parallel' on friday.

Reply:
If it is a port conflict issue, it could be a race condition like whether one test finishes while another is in a retry loop, freeing up a port or something. Do you happen to have any logs from one of your failing runs your could post here?

Reply:
```dc1-consul-server-0-tgnkjyh9l ~~ 2023-02-27T20:49:28.322Z [ERROR] agent.envoy: Error receiving new DeltaDiscoveryRequest; closing request channel: error="rpc error: code = Canceled desc = context canceled"
2023/02/27 14:49:28 Container is stopped id: 274fafc065fc image: c1e95f68-2e51-4ec0-97dc-49f12760e5bb:3e2ae1a9-c213-41d2-a840-f940152b6405
2023/02/27 14:49:28 Container is stopped id: 329a97d50aa3 image: 9a28a731-0720-4012-8978-3887fe037165:4081e5a3-fb70-47e6-abea-74a15f25c5b4
2023/02/27 14:49:28 Stopping container id: de55d358be1b image: docker.mirror.hashicorp.services/fortio/fortio
dc1-consul-server-0-g4nkx24tt ~~ 2023-02-27T20:49:28.750Z [WARN]  agent: Check socket connection failed: check=service:service1-sidecar-proxy:1 error="dial tcp 127.0.0.1:21000: connect: connection refused"
```